packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,51 @@
+When not working with an array as a source of data, an array of templates to
+render can be manually built, for example, by using a `for ... of` loop.
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+const templates = [];
```

Comment:
More stylistic question for @e111077: I had my snippets include the indentation needed so that after copy/paste you didn't manually indent them to make it look ok. But it does make the snippets look kinda weird since they're floating right more. Maybe something to decide and normalize once all the tutorials are committed.

Reply:
TBH I think it's fine removing the indents if it makes it look weird but i gotta say was nice for speed running your tutorial